[19.0][MIG] maintenance_plan

JasminSForgeFlow opened the pull request:
Standard Migration

@ForgeFlow

Depends on:

- [x] https://github.com/OCA/maintenance/pull/527

grindtildeath committed:
[ADD] maintenance_plan module (#2)

grindtildeath committed:
[ADD] maintenance_plan module (#2)

pedrobaeza committed:
[FIX] maintenance_plan: README try me on runbot link

pedrobaeza committed:
[FIX] maintenance_plan: README try me on runbot link

bestmazzo committed:
[MIG] maintenance_plan: Migration to 11.0

bestmazzo committed:
[MIG] maintenance_plan: Migration to 11.0

dalonsod committed:
[FIX] maintenance_plan: prevent preventive request cron crashing when equipment maintenance team is not filled

[ADD] icon.png

dalonsod committed:
[FIX] maintenance_plan: prevent preventive request cron crashing when equipment maintenance team is not filled

[ADD] icon.png

dalonsod committed:
[MIG] maintenance_plan: Migration to 12.0

[UPD] Update maintenance_plan.pot

dalonsod committed:
[MIG] maintenance_plan: Migration to 12.0

[UPD] Update maintenance_plan.pot

AdriaGForgeFlow committed:
[12.0][IMP] maintenance_plan

[ADD] instructions and notes to Plan

- Add menus for maintenance plans.
- Set active button on maintenance plans.

improve navigation to maintenance plans.

[ADD] post-migration script

AdriaGForgeFlow committed:
[12.0][IMP] maintenance_plan

[ADD] instructions and notes to Plan

- Add menus for maintenance plans.
- Set active button on maintenance plans.

improve navigation to maintenance plans.

[ADD] post-migration script

lromero-solvos committed:
Added translation using Weblate (Spanish)

lromero-solvos committed:
Added translation using Weblate (Spanish)

lromero-solvos committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (30 of 30 strings)

Translation: maintenance-12.0/maintenance-12.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-12-0/maintenance-12-0-maintenance_plan/es/

[UPD] README.rst

lromero-solvos committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (30 of 30 strings)

Translation: maintenance-12.0/maintenance-12.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-12-0/maintenance-12-0-maintenance_plan/es/

[UPD] README.rst

liweijie0812 committed:
Added translation using Weblate (Chinese (Simplified))

liweijie0812 committed:
Added translation using Weblate (Chinese (Simplified))

dalonsod committed:
[FIX] maintenance_plan: cron generated request should inherit equipment company

maintenance_plan 12.0.1.1.0

dalonsod committed:
[FIX] maintenance_plan: cron generated request should inherit equipment company

maintenance_plan 12.0.1.1.0

dalonsod committed:
[FIX] maintenance_plan: next_maintenance_date should be stored

dalonsod committed:
[FIX] maintenance_plan: next_maintenance_date should be stored

Commit:
[12.0][FIX] default equipment_id in plans

Commit:
[12.0][FIX] default equipment_id in plans

etobella committed:
[11.0][IMP] maintenance_plan: Return the created requests

etobella committed:
[11.0][IMP] maintenance_plan: Return the created requests

etobella committed:
[12.0][FIX] maintenance_project_plan

etobella committed:
[12.0][FIX] maintenance_project_plan

etobella committed:
[12.0][IMP] maintenance_plan: Add team on plans

[UPD] Update maintenance_plan.pot

[UPD] README.rst

maintenance_plan 12.0.3.0.0

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-12.0/maintenance-12.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-12-0/maintenance-12-0-maintenance_plan/

etobella committed:
[12.0][IMP] maintenance_plan: Add team on plans

[UPD] Update maintenance_plan.pot

[UPD] README.rst

maintenance_plan 12.0.3.0.0

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-12.0/maintenance-12.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-12-0/maintenance-12-0-maintenance_plan/

dalonsod committed:
[FIX] maintenance_plan: prevent False as display name when name is empty

[UPD] Update maintenance_plan.pot

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-12.0/maintenance-12.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-12-0/maintenance-12-0-maintenance_plan/

dalonsod committed:
[FIX] maintenance_plan: prevent False as display name when name is empty

[UPD] Update maintenance_plan.pot

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-12.0/maintenance-12.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-12-0/maintenance-12-0-maintenance_plan/

dalonsod committed:
Translated using Weblate (Spanish)

Currently translated at 98.4% (63 of 64 strings)

Translation: maintenance-12.0/maintenance-12.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-12-0/maintenance-12-0-maintenance_plan/es/

dalonsod committed:
Translated using Weblate (Spanish)

Currently translated at 98.4% (63 of 64 strings)

Translation: maintenance-12.0/maintenance-12.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-12-0/maintenance-12-0-maintenance_plan/es/

NuriaXifre committed:
[IMP] maintenance_plan: black, isort

NuriaXifre committed:
[IMP] maintenance_plan: black, isort

NuriaXifre committed:
[13.0][MIG] maintenance_plan

[UPD] Update maintenance_plan.pot

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-13.0/maintenance-13.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-13-0/maintenance-13-0-maintenance_plan/

NuriaXifre committed:
[13.0][MIG] maintenance_plan

[UPD] Update maintenance_plan.pot

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-13.0/maintenance-13.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-13-0/maintenance-13-0-maintenance_plan/

Commit:
[12.0][IMP] maintenance_plan: no equipment

[UPD] Update maintenance_plan.pot

maintenance_plan 13.0.1.1.0

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-13.0/maintenance-13.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-13-0/maintenance-13-0-maintenance_plan/

pre-commit update

Commit:
[12.0][IMP] maintenance_plan: no equipment

[UPD] Update maintenance_plan.pot

maintenance_plan 13.0.1.1.0

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-13.0/maintenance-13.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-13-0/maintenance-13-0-maintenance_plan/

pre-commit update

AaronHForgeFlow committed:
[FIX]maintenance_plan migration scripts not applicable in v13

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-13.0/maintenance-13.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-13-0/maintenance-13-0-maintenance_plan/

AaronHForgeFlow committed:
[FIX]maintenance_plan migration scripts not applicable in v13

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-13.0/maintenance-13.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-13-0/maintenance-13-0-maintenance_plan/

LoisRForgeFlow committed:
[13.0][IMP] maintenance_plan: make maintenance plan multi-company compatible.
includes migration scripts

LoisRForgeFlow committed:
[13.0][IMP] maintenance_plan: make maintenance plan multi-company compatible.
includes migration scripts

LoisRForgeFlow committed:
maintenance_plan: manual version bump

[UPD] Update maintenance_plan.pot

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-13.0/maintenance-13.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-13-0/maintenance-13-0-maintenance_plan/

LoisRForgeFlow committed:
maintenance_plan: manual version bump

[UPD] Update maintenance_plan.pot

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-13.0/maintenance-13.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-13-0/maintenance-13-0-maintenance_plan/

dalonsod committed:
[IMP] maintenance_plan: black, isort, prettier

dalonsod committed:
[IMP] maintenance_plan: black, isort, prettier

dalonsod committed:
[MIG] maintenance_plan: Migration to 14.0

[UPD] Update maintenance_plan.pot

[UPD] README.rst

dalonsod committed:
[MIG] maintenance_plan: Migration to 14.0

[UPD] Update maintenance_plan.pot

[UPD] README.rst

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (69 of 69 strings)

Translation: maintenance-14.0/maintenance-14.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-14-0/maintenance-14-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (69 of 69 strings)

Translation: maintenance-14.0/maintenance-14.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-14-0/maintenance-14-0-maintenance_plan/it/

etobella committed:
[FIX] maintenance_plan: Remake the computation of next date
* Use latest request_date both for done and todo searches.
* When there is a latest done apply interval

Co-authored-by: Lois Rilo <lois.rilo@forgeflow.com>
Co-authored-by: LoisRForgeFlow <LoisRForgeFlow@users.noreply.github.com>

maintenance_plan 14.0.1.0.1

etobella committed:
[FIX] maintenance_plan: Remake the computation of next date
* Use latest request_date both for done and todo searches.
* When there is a latest done apply interval

Co-authored-by: Lois Rilo <lois.rilo@forgeflow.com>
Co-authored-by: LoisRForgeFlow <LoisRForgeFlow@users.noreply.github.com>

maintenance_plan 14.0.1.0.1

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (69 of 69 strings)

Translation: maintenance-14.0/maintenance-14.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-14-0/maintenance-14-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (69 of 69 strings)

Translation: maintenance-14.0/maintenance-14.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-14-0/maintenance-14-0-maintenance_plan/it/

LoisRForgeFlow committed:
[IMP] maintenance_plan: next maintenance date tweaks

* only consider request after the start maintenance date.
* todo request ordered asc.
* add more tests.

maintenance_plan 14.0.1.0.2

LoisRForgeFlow committed:
[IMP] maintenance_plan: next maintenance date tweaks

* only consider request after the start maintenance date.
* todo request ordered asc.
* add more tests.

maintenance_plan 14.0.1.0.2

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (69 of 69 strings)

Translation: maintenance-14.0/maintenance-14.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-14-0/maintenance-14-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (69 of 69 strings)

Translation: maintenance-14.0/maintenance-14.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-14-0/maintenance-14-0-maintenance_plan/it/

BernatPForgeFlow committed:
[IMP] maintenance_plan: black, isort, prettier

BernatPForgeFlow committed:
[IMP] maintenance_plan: black, isort, prettier

BernatPForgeFlow committed:
[MIG] maintenance_plan: Migration to 15.0

BernatPForgeFlow committed:
[MIG] maintenance_plan: Migration to 15.0

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

mariadforgeflow committed:
[FIX] maintenance_plan: fix new request date when plan start date is later

mariadforgeflow committed:
[FIX] maintenance_plan: fix new request date when plan start date is later

OCA-git-bot committed:
maintenance_plan 15.0.1.0.1

OCA-git-bot committed:
maintenance_plan 15.0.1.0.1

MiquelRForgeFlow committed:
[FIX] maintenance_plan: avoid error if planning_step is not defined

Also, clean-up of useless from_string() calls.

MiquelRForgeFlow committed:
[FIX] maintenance_plan: avoid error if planning_step is not defined

Also, clean-up of useless from_string() calls.

mariadforgeflow committed:
[IMP] maintenance_plan: add chatter

mariadforgeflow committed:
[IMP] maintenance_plan: add chatter

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

OCA-git-bot committed:
maintenance_plan 15.0.1.1.0

OCA-git-bot committed:
maintenance_plan 15.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

OCA-git-bot committed:
maintenance_plan 15.0.1.1.1

OCA-git-bot committed:
maintenance_plan 15.0.1.1.1

sergiocorato committed:
[15.0][FIX] maintenance_plan generate requests for inactive equipment

sergiocorato committed:
[15.0][FIX] maintenance_plan generate requests for inactive equipment

OCA-git-bot committed:
maintenance_plan 15.0.1.1.2

OCA-git-bot committed:
maintenance_plan 15.0.1.1.2

mymage committed:
Translated using Weblate (Italian)

Currently translated at 67.9% (72 of 106 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 67.9% (72 of 106 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (106 of 106 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (106 of 106 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
[REF] moving function get_relativedelta to method

mymage committed:
[REF] moving function get_relativedelta to method

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 58.4% (62 of 106 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/pt/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 58.4% (62 of 106 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/pt/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (106 of 106 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/pt/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (106 of 106 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/pt/

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

OCA-git-bot committed:
maintenance_plan 15.0.1.2.0

OCA-git-bot committed:
maintenance_plan 15.0.1.2.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

victoralmau committed:
[IMP] maintenance_plan: Set planned_hours in request from plan if maintenance_timesheet is installed.

victoralmau committed:
[IMP] maintenance_plan: Set planned_hours in request from plan if maintenance_timesheet is installed.

kluna1998 committed:
[FIX] maintenance_plan: fix maintence request form view

kluna1998 committed:
[FIX] maintenance_plan: fix maintence request form view

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (111 of 111 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

victoralmau committed:
[IMP] maintenance_plan: Add Generate requests for current threshold button from plan view form.

Add button from maintenance plan to allow generate maintenance requests
without waiting for cron.

TT41729

victoralmau committed:
[IMP] maintenance_plan: Add Generate requests for current threshold button from plan view form.

Add button from maintenance plan to allow generate maintenance requests
without waiting for cron.

TT41729

victoralmau committed:
[IMP] maintenance_plan: Add instructions to request report.

TT41671

victoralmau committed:
[IMP] maintenance_plan: Add instructions to request report.

TT41671

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

OCA-git-bot committed:
maintenance_plan 15.0.1.3.0

OCA-git-bot committed:
maintenance_plan 15.0.1.3.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

OCA-git-bot committed:
maintenance_plan 15.0.1.4.0

OCA-git-bot committed:
maintenance_plan 15.0.1.4.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

OCA-git-bot committed:
maintenance_plan 15.0.1.5.0

OCA-git-bot committed:
maintenance_plan 15.0.1.5.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

OCA-git-bot committed:
maintenance_plan 15.0.1.5.1

OCA-git-bot committed:
maintenance_plan 15.0.1.5.1

victoralmau committed:
[IMP] maintenance_plan: Change tests to allow inheritance

TT41735

victoralmau committed:
[IMP] maintenance_plan: Change tests to allow inheritance

TT41735

victoralmau committed:
[IMP] maintenance_plan: Skip assigned mail + Activity mail of the requests created from plans

TT41870

victoralmau committed:
[IMP] maintenance_plan: Skip assigned mail + Activity mail of the requests created from plans

TT41870

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

OCA-git-bot committed:
maintenance_plan 15.0.1.6.0

OCA-git-bot committed:
maintenance_plan 15.0.1.6.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

OCA-git-bot committed:
maintenance_plan 15.0.1.6.1

OCA-git-bot committed:
maintenance_plan 15.0.1.6.1

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

victoralmau committed:
[IMP] maintenance_plan: Add rules to maintenance.plan to view records of the equipments they follow

TT42535

victoralmau committed:
[IMP] maintenance_plan: Add rules to maintenance.plan to view records of the equipments they follow

TT42535

OCA-git-bot committed:
maintenance_plan 15.0.1.7.0

OCA-git-bot committed:
maintenance_plan 15.0.1.7.0

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 85.9% (98 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 85.9% (98 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/es/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

etobella committed:
[IMP] maintenance_plan: Allow to generate generic plans for multiple equipments defined with a domain

etobella committed:
[IMP] maintenance_plan: Allow to generate generic plans for multiple equipments defined with a domain

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (114 of 114 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/es/

etobella committed:
[FIX] maintenance_plan: The search must work with newIds

etobella committed:
[FIX] maintenance_plan: The search must work with newIds

etobella committed:
[IMP] maintenance_plan: Add Team filter

etobella committed:
[IMP] maintenance_plan: Add Team filter

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

victoralmau committed:
[IMP] maintenance_plan: Add freeze_time to prevent errors in tests

victoralmau committed:
[IMP] maintenance_plan: Add freeze_time to prevent errors in tests

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

etobella committed:
[FIX] maintenance_plan: adapt tests

etobella committed:
[FIX] maintenance_plan: adapt tests

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (118 of 118 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (118 of 118 strings)

Translation: maintenance-15.0/maintenance-15.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-15-0/maintenance-15-0-maintenance_plan/it/

alexeirivera87 committed:
[MIG] maintenance_plan: Migration to 16.0

alexeirivera87 committed:
[MIG] maintenance_plan: Migration to 16.0

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (118 of 118 strings)

Translation: maintenance-16.0/maintenance-16.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-16-0/maintenance-16-0-maintenance_plan/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (118 of 118 strings)

Translation: maintenance-16.0/maintenance-16.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-16-0/maintenance-16-0-maintenance_plan/es/

FernandoRomera committed:
[IMP] maintenance_plan: pre-commit auto fixes

FernandoRomera committed:
[IMP] maintenance_plan: pre-commit auto fixes

FernandoRomera committed:
[MIG] maintenance_plan: Migration to 17.0

FernandoRomera committed:
[MIG] maintenance_plan: Migration to 17.0

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-17.0/maintenance-17.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-17-0/maintenance-17-0-maintenance_plan/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-17.0/maintenance-17.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-17-0/maintenance-17-0-maintenance_plan/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (115 of 115 strings)

Translation: maintenance-17.0/maintenance-17.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-17-0/maintenance-17-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (115 of 115 strings)

Translation: maintenance-17.0/maintenance-17.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-17-0/maintenance-17-0-maintenance_plan/it/

vincent-hatakeyama committed:
[FIX] maintenance_plan: fix the use of id in act_window

Without this fix, the smart button using the act_window fails with an
error.

vincent-hatakeyama committed:
[FIX] maintenance_plan: fix the use of id in act_window

Without this fix, the smart button using the act_window fails with an
error.

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (115 of 115 strings)

Translation: maintenance-17.0/maintenance-17.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-17-0/maintenance-17-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (115 of 115 strings)

Translation: maintenance-17.0/maintenance-17.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-17-0/maintenance-17-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (115 of 115 strings)

Translation: maintenance-17.0/maintenance-17.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-17-0/maintenance-17-0-maintenance_plan/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (115 of 115 strings)

Translation: maintenance-17.0/maintenance-17.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-17-0/maintenance-17-0-maintenance_plan/it/

carlos-lopez-tecnativa committed:
[IMP] base_maintenance: Fine-tuning of the form view
after this commit https://github.com/odoo/odoo/commit/3bd841086d92246e55771785bcb69501af712ab6 the maintenance request form view now has an independent page. This commit adapts the view to align with the new page structure.

carlos-lopez-tecnativa committed:
[IMP] base_maintenance: Fine-tuning of the form view
after this commit https://github.com/odoo/odoo/commit/3bd841086d92246e55771785bcb69501af712ab6 the maintenance request form view now has an independent page. This commit adapts the view to align with the new page structure.

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-17.0/maintenance-17.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-17-0/maintenance-17-0-maintenance_plan/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: maintenance-17.0/maintenance-17.0-maintenance_plan
Translate-URL: https://translation.odoo-community.org/projects/maintenance-17-0/maintenance-17-0-maintenance_plan/

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

victoralmau committed:
[IMP] maintenance_plan: Define module as deprecated in readme

https://github.com/OCA/maintenance/pull/414#issuecomment-2668929617

victoralmau committed:
[IMP] maintenance_plan: Define module as deprecated in readme

https://github.com/OCA/maintenance/pull/414#issuecomment-2668929617

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

ypapouin committed:
[IMP] maintenance_plan: Remove "deprecated" status

ypapouin committed:
[IMP] maintenance_plan: Remove "deprecated" status

ypapouin committed:
[IMP] maintenance_plan: Path added to actions

ypapouin committed:
[IMP] maintenance_plan: Path added to actions

ypapouin committed:
[FIX] maintenance_plan: Odoo dependency flaw

ypapouin committed:
[FIX] maintenance_plan: Odoo dependency flaw

ypapouin committed:
[IMP] maintenance_plan: pre-commit auto fixes

ypapouin committed:
[IMP] maintenance_plan: pre-commit auto fixes

ypapouin committed:
[MIG] maintenance_plan: Migration to 18.0

ypapouin committed:
[MIG] maintenance_plan: Migration to 18.0

ypapouin committed:
[IMP] maintenance_plan: Replace note with instructions

- `maintenance.request` views are now set in their own file
- `note` field replaced with existing combo `instruction_type,instruction_pdf,instruction_google_slide,instruction_text`

ypapouin committed:
[IMP] maintenance_plan: Replace note with instructions

- `maintenance.request` views are now set in their own file
- `note` field replaced with existing combo `instruction_type,instruction_pdf,instruction_google_slide,instruction_text`

ypapouin committed:
[IMP] maintenance_plan: Setting added to install "maintenance_plan_only" module

ypapouin committed:
[IMP] maintenance_plan: Setting added to install "maintenance_plan_only" module

Commit:
[UPD] Update maintenance_plan.pot

Commit:
[UPD] Update maintenance_plan.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

JasminSForgeFlow committed:
[IMP] maintenance_plan: pre-commit auto fixes

JasminSForgeFlow committed:
[IMP] maintenance_plan: pre-commit auto fixes

JasminSForgeFlow committed:
[MIG] maintenance_plan: Migration to 19.0

JasminSForgeFlow committed:
[MIG] maintenance_plan: Migration to 19.0

AaronHForgeFlow commented:
Hi @JasminSForgeFlow, dependency is merged. Can you please rebase and update the main PR comment? Thanks

JasminSForgeFlow commented:
> Hi @JasminSForgeFlow, dependency is merged. Can you please rebase and update the main PR comment? Thanks

Done, Thanks

AaronHForgeFlow commented on maintenance_plan/tests/common.py:
```diff
@@ -0,0 +1,80 @@
+        cls.maintenance_plan_obj = cls.env["maintenance.plan"]
+        cls.maintenance_equipment_obj = cls.env["maintenance.equipment"]
+        cls.cron = cls.env.ref("maintenance_plan.maintenance_requests_cron")
+        cls.weekly_kind = cls.env.ref("maintenance_plan.maintenance_kind_weekly")
```
tests wont load demo data, you have to create the record

AaronHForgeFlow reviewed (commented):
Functional + code ok, just pending to fix tests.

Quan-nhm reviewed (approved):
Code review LGTM

yvaucher commented:
/ocabot migration maintenance_plan

yvaucher commented:
Test still needs to be fixed

yvaucher commented on maintenance_plan/models/maintenance_plan.py:
```diff
@@ -0,0 +1,264 @@
+    )
+
+    @api.model
+    def _search_search_equipment(self, operator, value):
```
Locally this method fails when opening the equipment view, can you add a test on it?

It fails when called by the `_compute_search_maintenance_plan_count`

EDIT: We have to support the `in` operator. It seems to not recognize the operator because 

` .search_count([("search_equipment_id", "=", equipment.id)])`

translates to `in recordset`

Args of `_search_search_equipment`

```
self = maintenance.plan()
operator = 'in'
value = OrderedSet([758])
```

yvaucher replied:
You will find a fix here https://github.com/ForgeFlow/maintenance/pull/1

JasminSForgeFlow replied:
@ypapouin Thanks for the fix, I have merged that. There was an issue with pre-commit, which I fixed, and your tests were incorrect, so I have removed that one; it has multiple code issues.

yvaucher committed:
[FIX] maintenance_plan: search of search_equipment field

The ORM optimize the domain by replacing `=` by `in`thus we need to support the in operator.

Co-authored-by: Jasmin Solanki <jasmin.solanki@forgeflow.com>"

BhaveshHeliconia reviewed (approved):
Code review LGTM!

yvaucher commented:
/ocabot merge nobump
